ase/src/main/java/com/ase/ase/activemq/UpdateDataMessage.java

```diff
@@ -0,0 +1,33 @@
+package com.ase.ase.activemq;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+
+import java.io.Serializable;
+
+@JsonSerialize
+public class UpdateDataMessage implements Serializable {
+
+  private boolean update;
+
+  public UpdateDataMessage(boolean update) {
+    this.update = update;
+  }
+
+  @JsonProperty("update")
+  public boolean update() {
+    return update;
+  }
+
+  @Override
+  public String toString() {
+    try {
+      return new ObjectMapper().writeValueAsString(this);
+    } catch (JsonProcessingException e) {
+      e.printStackTrace();
+    }
+    return null;
+  }
```

Comment:
Why do you not write your own String for this one class-variable? (would look easier than this)

Reply:
I could do that but i thought that writing a hardcoded json string in json format is not that beautiful. The string would be: "{\"update\":" + update + "}".

Reply:
Maybe rename it to toJsonString so that it is clear what this method is for

Reply:
I added another method called "toJson" which is called in the toString method. The toString is needed because in the messaging service, we expect an object of type "Serializable" and the toString of this object is called to get the string which is written to the stream.